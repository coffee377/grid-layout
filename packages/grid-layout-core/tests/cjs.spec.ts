const core = require('grid-layout-core');
const demo = require('./demo').default;

test('dom code', () => {
  const code = core.domCodeText(demo, true, true);
  expect(code).toMatchSnapshot();
});

test('css code', () => {
  const code = core.cssCodeText(demo, true, {});
  expect(code).toMatchSnapshot();
});
