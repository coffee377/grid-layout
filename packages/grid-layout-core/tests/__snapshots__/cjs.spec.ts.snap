// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`css code 1`] = `
".demo1 {
  display: grid;
  grid-template-columns: 3fr 4fr 3fr;
  grid-template-rows: repeat(8,55px);
  grid-row-gap: 30px;
  grid-column-gap: 30px;
  width: 90vw;
  height: calc(100vh - 30px);
  margin: 30px auto 0;
}

.item {
  padding: 5px;
  border: solid #0b5d9c 1px;
  border-radius: 8px;
}

.item1 {
  grid-area: 1 / 1 / span 4 / span 1;
}

.item2 {
  grid-area: 5 / 1 / span 4 / span 1;
}

.item3 {
  grid-area: 1 / 2 / span 5 / span 1;
}

.item4 {
  grid-area: 6 / 2 / span 3 / span 1;
}

.item5 {
  grid-area: 1 / 3 / span 4 / span 1;
}

.item6 {
  grid-area: 5 / 3 / span 4 / span 1;
}"
`;

exports[`dom code 1`] = `
"<div style=\\"display:grid;grid-template-columns:3fr 4fr 3fr;grid-template-rows:repeat(8,55px);grid-row-gap:30px;grid-column-gap:30px;width:90vw;height:calc(100vh - 30px);margin:30px auto 0;\\">
	<div style=\\"padding:5px;border:solid #0b5d9c 1px;border-radius:8px;grid-area:1 / 1 / span 4 / span 1;\\"></div>
	<div style=\\"padding:5px;border:solid #0b5d9c 1px;border-radius:8px;grid-area:5 / 1 / span 4 / span 1;\\"></div>
	<div style=\\"padding:5px;border:solid #0b5d9c 1px;border-radius:8px;grid-area:1 / 2 / span 5 / span 1;\\"></div>
	<div style=\\"padding:5px;border:solid #0b5d9c 1px;border-radius:8px;grid-area:6 / 2 / span 3 / span 1;\\"></div>
	<div style=\\"padding:5px;border:solid #0b5d9c 1px;border-radius:8px;grid-area:1 / 3 / span 4 / span 1;\\"></div>
	<div style=\\"padding:5px;border:solid #0b5d9c 1px;border-radius:8px;grid-area:5 / 3 / span 4 / span 1;\\"></div>
</div>"
`;
